### PR TITLE
Update advanced development

### PR DESCRIPTION
**Pull request by DylanVerstraete**

### CODE REVIEW

File: src/3bot_sysadmin/admin_tools/3bot_farm_mgmt.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ This section of your 3Bot lets you create, manage and monitor your farms.
 
 ## Creating a new farm
 
-Follow this detailed guide to [create a farm](../../farming/farm_init.md)
+Follow this detailed guide to [create a farm](farm_init.md)
```
Comment (DylanVerstraete) [inline src/3bot_sysadmin/admin_tools/3bot_farm_mgmt.md:9]: end the sentence with a `.`

File: src/develop/code/container/code_container.md [hunk 2]
```diff
@@ -14,9 +14,9 @@ The aim is to deploy a simple container using an Flist which is further describe
 Each overlay network is private and contains private IP addresses. Each overlay network is deployed in such a way that is has no connection to the public (IPv4 or IPv6) network directly. In order to work with such a network a tunnel needs to be created between the overlay network on the grid and your local network. You could find instructions how to create a network [here](code_network.md)
 
 
-#### What is a Flist? 
+#### What is a Flist?
```
Comment (DylanVerstraete) [inline src/develop/code/container/code_container.md:17]: a = an

File: src/develop/code/container/code_container.md [hunk 3]
```diff
@@ -14,9 +14,9 @@ The aim is to deploy a simple container using an Flist which is further describe
...
+A Flist is a very special container image. One of the challenges with industry-leading technologies like Docker and Kubernetes is that every node involved in an IT architecture has to have local copies of all of the images it needs to run as containers. These could either be base images on which specific modifications need to be made or they are specific images downloaded from the docker hub or a private image repository (enterprise use cases). Having these images exists on many different nodes requires these to be downloaded and maintained for version control and bug fixes. This is wasteful (many times the same image required storage space) and time-consuming.
```
Comment (DylanVerstraete) [inline src/develop/code/container/code_container.md:19]: An Flist

File: src/develop/code/container/code_container.md [hunk 4]
```diff
@@ -60,11 +60,11 @@ zos.container.create(node_id={string},    # node_id to deploy the Flist
      network_name={string},  # network_name deployed on the node (node could have multiple private networks)
      ip_address={string},   # one IP address in the range of the chosen network_name on the node
      Flist={string},    # Flist of the container you want to install, htttp hub location.
```
Comment (DylanVerstraete) [inline src/develop/code/container/code_container.md:62]: without caps = flist

File: src/develop/code/network/code_network.md [hunk 5]
```diff
@@ -9,11 +9,11 @@ Please check the [general requirements](code.md)
 
 #### 1. Load required libraries and create empty reservation structure
 
-To be able to made a reservation we need to local the System Abstraction Layer (SAL) and create a [capacity pool](code_pool.md)
+To be able to make a reservation we need to create a [capacity pool](code_pool.md)
```
Comment (DylanVerstraete) [inline src/develop/code/network/code_network.md:12]: end sentence with a .

File: src/develop/code/network/code_network.md [hunk 6]
```diff
@@ -48,11 +48,11 @@ for i, node in enumerate(pool.node_ids):
...
+Please store the list of nodes somewhere for you reference to deploy containers and architectures, or you could write code to store this to a file. All nodes are connected with IPv6 to the internet. If you have IPv6 at home you could create a wireguard configuration to any of the nodes as they all speak IPv6 and any node could act as you private gateway into your overlay network. If you do not have IPv6 at home you need to identify a nodes that has IPv4 capabilities. In this example the nodes in Belgium have a dual-stack and could therefore be used to provide an IPv4 gateway into your overlay network.
```
Comment (DylanVerstraete) [inline src/develop/code/network/code_network.md:51]: Replace all `could` with `can`

**Comment by Sacha96** [comment]
Good for me @DylanVerstraete

**Comment by xmonader** [comment]
changes have been manually picked and applied on development branch.

**Pull request closed without merging**